x/gov/types/codec.go

```diff
@@ -27,6 +27,7 @@ func RegisterProposalTypeCodec(o interface{}, name string) {
 	ModuleCdc.RegisterConcrete(o, name, nil)
 }
 
+// TODO determine a good place to seal this codec
```

Comment:
fix todo or create issue

Reply:
issue created for this https://github.com/cosmos/cosmos-sdk/issues/4473